context/auth.js

```diff
@@ -22,11 +22,10 @@ export const AuthProvider = ({ children }) => {
         provider.addScope('profile');
         firebase.auth().signInWithPopup(provider)
             .then(() => {
-                //console.log("firebase login success")
+            //Notify to user by succes login    
```

Comment:
Adding a modal would be a good option here

Reply:
Ya, a small modal to show the user has logged in/signed up successfully would be great.